install.sh

```diff
@@ -34,14 +34,15 @@
 # ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-# 
+#
 # The views and conclusions contained in the software and documentation are those
 # of the authors and should not be interpreted as representing official policies,
 # either expressed or implied, of the FreeBSD Project.
 
 get_latest_release() {
-  echo "dev"
-  # curl --silent "https://wasmtime.sh/latest-version"
+  curl --silent "https://api.github.com/repos/bytecodealliance/wasmtime/releases/latest" | \
+    grep tag_name | \
+    cut -d '"' -f 4
```

Comment:
`| jq .tag_name` maybe to avoid dependence on JSON formatting?

Reply:
I think one of the purposes of this script is to run in many contexts and `jq` I don't think it as ubiquitous as `cut`, but then again I never know what `cut` is myself and I'm just [blindly following StackOverflow](https://stackoverflow.com/questions/24987542/is-there-a-link-to-github-for-downloading-a-file-in-the-latest-release-of-a-repo)

Reply:
Ah that's a good point, for some reason I had been thinking this runs in a CI context but actually it runs on arbitrary end-user systems... so I agree, `grep` and `cut` are the better choice here.